packages/form-core/src/FormApi.ts

```diff
@@ -234,7 +236,7 @@ export type FormState<TFormData> = {
   /**
    * The error map for the form itself.
    */
-  errorMap: ValidationErrorMap
+  errorMap: ServerValidationErrorMap
```

Comment:
I'm not fully convinced this should be called `Server` since we're using it client side too, but I couldn't come up with a better name so I'm just leaving a note here :D

Maybe `Detailed` or `Expanded`? I don't know

Reply:
On refection; terrible name. Let's call it `FormValidationErrorMap`